payroll/models/hr_payslip.py

```diff
@@ -653,11 +670,12 @@ def _sum_salary_rule_category(localdict, category, amount):
                     result_dict[key] = {
                         "salary_rule_id": rule.id,
                         "contract_id": contract.id,
-                        "name": rule.name,
+                        "name": computed_name != "False" and computed_name or rule.name,
```

Comment:
Should this be False or "False"?

Reply:
The method do a str(result_name) and that returns a "False" string, that's why the condition takes both cases if it is False or "False".